examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,10 +5,9 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
+n_subjects = 10  # Number of subjects to consider for group-sparse covariance
```

Comment:
Usually the number of subjects is chosen to have an exmaple running in a reasonable time with reasonable good results.